Sources/HaCWebsiteLib/Models/NewWorkshop.swift

```diff
@@ -0,0 +1,54 @@
+import Foundation
+import Yaml
+
+/// Represents a workshop
+// TODO: Rename this struct to `Workshop` when it's ready for use
+public struct NewWorkshop {
+  let title: String
+
+  /// People that have directly contributed content to this workshop
+  let contributors: [String]
+
+  /// Non-contributors we would like to thank for helping to inspire, develop or facilitate this workshop
+  let thanks: [String]
+
+  /// Links that attendees may which to read after the workhsop
+  let furtherReadingLinks: [(String, URL)]
+
+  /// A link to a video recording of this workshop
+  let recordingLink: URL?
+
+  /// A link to any slides for this workshop
+  let slidesLink: URL?
+
+  /// A set of keywords associated with the workshop
+  let tags: [String]
+
+  /// A string of a https://spdx.org/licenses/ license identifier
+  // TODO: Make a type safe version of this
+  let license: String
```

Comment:
@jaredkhan Is it ok if I get on this and push it to this PR?

Reply:
Let’s put it in a separate PR after this goes in since this blocks things

Reply:
Also, I reckon this can go in a separate module @moosichu so that we can move it to a separate repo eventually and other people can use it for their zesty projects 😊

Reply:
created #166 :)